src/index.tsx

```diff
@@ -4,16 +4,12 @@ import { HashRouter as Router } from 'react-router-dom';
 
 import { store } from './app/store';
 import { App } from './App';
-import { UsersProvider } from './components/UsersContext';
 
 const Root = () => (
```

Comment:
It is recommended to use arrow function instead of function declaration for functional components.